### PR TITLE
Fix executable not found error

### LINKED ISSUE #17: Include `rev` keyword pointing to latest version in README snippet
## Problem statement

The snippet in the README is missing the `rev`  keyword, as a new user of pre-commit, it would throw me an error `missing required key: rev` which would be hard to resolve if i didn't know `rev` is to be pointed to a version.

Can we update the documentation with the fix?
```
---
repos:
  - repo: https://github.com/thoth-station/test-thoth-pre-commit-hook
    rev: v0.1.0
    hooks:
      - id: thoth-advise
        args: ["--recommendation-type", "security"]
````

### PR DESCRIPTION
**Pull request by mayaCostantini**
## This introduces a breaking change

- No

## This should yield a new module release

- Yes

## This Pull Request implements

Fix error caused by executable entrypoint not found.
Also closes #17

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

File: setup.py [hunk 1]
```diff
@@ -27,10 +27,14 @@ def get_version():
 VERSION = get_version()
 setup(
     name="thoth_pre_commit_hook",
-    entry_points={"console_scripts": ["thamos=thamos.cli:cli"]},
+    entry_points={
+        "console_scripts": [
+            "thoth-advise = thoth.thoth_pre_commit_hook.thoth_advise:main"
```
Comment (harshad16) [inline setup.py:32]: As we discuss over the chat this doesnt fucntion as required.
Reason being https://packaging.python.org/en/latest/specifications/entry-points/, entry point looks for a format of `module.object`
and `thoth.thoth_pre_commit_hook` is being intrepret as module, where only `thoth_pre_commit_hook` is the module.

we can change this in the following way:

```
entry_points={
        "console_scripts": ["thoth-advise = thoth_pre_commit_hook:thoth_advise.main"]
    },
packages=["thoth_pre_commit_hook"],
package_dir={
        "thoth_pre_commit_hook": "./thoth/thoth_pre_commit_hook",
    },
```
here we are making setup py aware explicitly that `thoth_pre_commit_hook` is the module 
and we would like to run the file from this module

**Review by harshad16 — CHANGES_REQUESTED** [review]
very great work
here is the suggestion for the error we are facing

**Comment by mayaCostantini** [comment]
Thanks for the review @harshad16, this should be good now. Feel free to test with the last revision though :+1:

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

**Pull request merged**